docs/en/observability/tab-widgets/uptime-monitoring/run-test-configure-content.asciidoc

```diff
@@ -0,0 +1,133 @@
+:synthetics_version: v1.0.0-beta.26
+
+// tag::managed[]
+
+beta[] Configure each monitor directly in your test code using `monitor.use`.
+You must configure the monitor for each journey, which allows you to set different options for each monitor.
```

Comment:
We should also mention that if the users want to have a default schedule and locations for all monitors, they would need to configure them on `synthetics.config.ts` file. Ex - https://github.com/elastic/synthetics/blob/b443c258788434d12c53c24bd711fc65f2c16668/templates/synthetics.config.ts#L11-L18

Reply:
Are `locations` and `schedule` the only two options that can be configured across all monitors in the config file or were those just examples?

Reply:
Answered below!